welcome/ocp-overview.adoc

```diff
@@ -0,0 +1,33 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ocp-overview"]
+= OpenShift Container Platform overview
```

Comment:
This title should match what is in topic_map. 
```suggestion
=  Introduction to OpenShift Container Platform
```
Or change the topic map to match _OpenShift Container Platform overview_.

Reply:
good catch. This is basically just a reorg and pulled out from another book.